README.md

```diff
@@ -129,9 +126,55 @@ If this process sounds familiar, it's because it's basically the same as having
 
 - **JQuery** Anytime you see that money sign, you're working with a Jquery object. That Jquery object is set by referring to a CSS selector for example $('body') to select the body element of the DOM.
 
+### Questions from Homework
+**What parts of the page does the ApplicationView directly manage?**
+
+The ApplicationView manages the "New Contact" form.
+
+**What parts of the page does the RolodexView directly manage?**
+
+The RolodexView manages the list of contacts and the details modal. It also listens for click events to hide the modal.
+
+**What parts of the page does the ContactView directly manage?**
+
+A ContactView manages a single contact card.
+
+**Which views are rendered when the RolodexCollection has been updated?**
+
+The RolodexView and all ContactViews.
+
+**For all custom events, list the following:**
+
+1. Which view triggers the event
+1. Which view handles/listens for the event
+
+We'll include all events here, not just custom ones.
+
+| Event                    | Type               | What Triggers | What Handles/Listens |
+|:-------------------------|:-------------------|:--------------|:---------------------|
+| Click on `Save` button   | DOM                | The user      | ApplicationView      |
+| Click on `Clear` button  | DOM                | The user      | ApplicationView      |
+| Click on a contact card  | DOM                | The user      | ContactView          |
+| Click off a contact card | DOM                | The user      | RolodexView          |
+| `update` the Rolodex     | Backbone Automatic | Backbone when the collection is updated. Updates come from ApplicationView (add a contact) | RolodexView |
```

Comment:
I think we should specify that Rolodex itself triggers this event. While it's accurate that Backbone's code for updating a collection is what ultimately triggers the event, I think it's also important to get across that the _object_ that triggers it is the Rolodex collection in particular.